### PR TITLE
Update Translators Table

### PR DESCRIPTION
**Pull request by osama24sy**
related to [#835](https://github.com/python/devguide/issues/835)

### CODE REVIEW

File: documenting.rst [hunk 1]
```diff
@@ -1655,15 +1655,15 @@ in production; others are works in progress.
...
-.. _bpo_mattwang44: https://bugs.python.org/user39654
+.. _bpo_cocoatomo: https://github.com/cocoatomo
+.. _bpo_gbtami: https://github.com/gbtami
+.. _bpo_kushal: https://github.com/Kushal997-das
+.. _bpo_mdk: https://github.com/JulienPalard
+.. _bpo_mmdbalkhi: https://github.com/mmdbalkhi
+.. _bpo_oonid: https://github.com/oonid
+.. _bpo_osdotsystem: https://github.com/Abdur-rahmaanJ
+.. _bpo_zhsj: https://github.com/zhsj
+.. _bpo_mattwang44: https://github.com/mattwang44
```
Comment (ezio-melotti) [inline documenting.rst:1666]: I would also rename the links from `_bpo_*` to `_gh_*`, here and in the table above.

**Review by ezio-melotti — CHANGES_REQUESTED** [review]

File: documenting.rst (hunk 1 above)
Comment (osama24sy) [reply documenting.rst]: Done

**Review by ezio-melotti — APPROVED** [review]

File: documenting.rst [hunk 2]
```diff
@@ -1596,35 +1596,35 @@ in production; others are works in progress.
...
 | Arabic (ar)     | `Abdur-Rahmaan Janhangeer     | `GitHub <github_ar_>`_     |
-|                 | <bpo_osdotsystem_>`_          |                            |
+|                 | <gh_osdotsystem_>`_           |                            |
 +-----------------+-------------------------------+----------------------------+
-| Bengali as      | `Kushal Das <bpo_kushal_>`_   | `GitHub <github_bn_in_>`_  |
+| Bengali as      | `Kushal Das <gh_kushal_>`_    | `GitHub <github_bn_in_>`_  |
 | spoken in       |                               |                            |
 | India (bn_IN)   |                               |                            |
 +-----------------+-------------------------------+----------------------------+
 | French (fr)     | `Julien Palard (mdk)          | `GitHub <github_fr_>`_     |
```
Comment (ezio-melotti) [inline documenting.rst:1605]:
```suggestion
| French (fr)     | `Julien Palard (@JulienPalard)| `GitHub <github_fr_>`_     |
```

File: documenting.rst [hunk 3]
```diff
@@ -1596,35 +1596,35 @@ in production; others are works in progress.
...
-|                 | <bpo_mdk_>`_                  |                            |
+|                 | <gh_mdk_>`_                   |                            |
 +-----------------+-------------------------------+----------------------------+
 | Hindi as spoken |                               | `GitHub <github_hi_in_>`_  |
 | in India (hi_IN)|                               |                            |
 +-----------------+-------------------------------+----------------------------+
 | Hungarian (hu)  | `Tamás Bajusz (gbtami)        | `GitHub <github_hu_>`_     |
```
Comment (ezio-melotti) [inline documenting.rst:1611]:
```suggestion
| Hungarian (hu)  | `Tamás Bajusz (@gbtami)       | `GitHub <github_hu_>`_     |
```

File: documenting.rst [hunk 4]
```diff
@@ -1596,35 +1596,35 @@ in production; others are works in progress.
...
+|                 | <gh_gbtami_>`_                | `Mailing List <list_hu_>`_ |
 +-----------------+-------------------------------+----------------------------+
-| Indonesian (id) | `Oon Arfiandwi <bpo_oonid_>`_ | `GitHub <github_id_>`_     |
+| Indonesian (id) | `Oon Arfiandwi <gh_oonid_>`_  | `GitHub <github_id_>`_     |
 +-----------------+-------------------------------+----------------------------+
 | Italian (it)    |                               | `mail <mail_it_>`_         |
 +-----------------+-------------------------------+----------------------------+
 | Japanese (ja)   | `Kinebuchi Tomohiko           | `GitHub <github_ja_>`_     |
-|                 | (cocoatomo) <bpo_cocoatomo_>`_| `Doc <doc_ja_>`_           |
+|                 | (cocoatomo) <gh_cocoatomo_>`_ | `Doc <doc_ja_>`_           |
```
Comment (ezio-melotti) [inline documenting.rst:1619]:
```suggestion
|                 | (@cocoatomo) <gh_cocoatomo_>`_| `Doc <doc_ja_>`_           |
```

File: documenting.rst [hunk 5]
```diff
@@ -1596,35 +1596,35 @@ in production; others are works in progress.
...
 +-----------------+-------------------------------+----------------------------+
 | Korean (ko)     |                               | `GitHub <github_ko_>`_     |
 |                 |                               | `Doc <doc_ko_>`_           |
 +-----------------+-------------------------------+----------------------------+
 | Lithuanian (lt) |                               | `mail <mail_lt_>`_         |
 +-----------------+-------------------------------+----------------------------+
 | Persian (fa)    | `Komeil Parseh (mmdbalkhi)    | `GitHub <github_fa_>`_     |
```
Comment (ezio-melotti) [inline documenting.rst:1626]:
```suggestion
| Persian (fa)    | `Komeil Parseh (@mmdbalkhi)   | `GitHub <github_fa_>`_     |
```

File: documenting.rst [hunk 6]
```diff
@@ -1640,30 +1640,30 @@ in production; others are works in progress.
...
 | (zh-cn)         |                               | `Doc <doc_zh_cn_>`_        |
 +-----------------+-------------------------------+----------------------------+
 | Spanish (es)    | Raúl Cumplido                 | `GitHub <github_es_>`_     |
 +-----------------+-------------------------------+----------------------------+
 | Traditional     | `王威翔 Matt Wang             | `GitHub <github_zh_tw_>`_  |
-| Chinese         | <bpo_mattwang44_>`_,          | `Transifex <tx_zh_tw_>`_   |
+| Chinese         | <gh_mattwang44_>`_,           | `Transifex <tx_zh_tw_>`_   |
 | (zh-tw)         | Josix Wang                    | `Doc <doc_zh_tw_>`_        |
 +-----------------+-------------------------------+----------------------------+
 | Turkish (tr)    | `Ege Akman (egeakman)         | `GitHub <github_tr_>`_     |
```
Comment (ezio-melotti) [inline documenting.rst:1653]:
```suggestion
| Turkish (tr)    | `Ege Akman (@egeakman)        | `GitHub <github_tr_>`_     |
```

**Pull request merged**